src/middlewares/loginThunk.ts

```diff
@@ -0,0 +1,21 @@
+import {Dispatch} from 'redux';
+
+import {logOut, setAuth} from '../actions/loginActions';
+import {authentication} from '../services/storage/authentication';
+
+export const tryLogin =
+  (credentials: {email: string; password: string}) => (dispatch: Dispatch) => {
+    const result =
```

Comment:
result is not valid var name for it, this condition meant that is user enter valid credentials or not ... as I understand